code/model/modelinterp.cpp

```diff
@@ -1793,6 +1882,10 @@ void parse_bsp(int offset, ubyte *bsp_data)
 			case OP_BOUNDBOX:
 				break;
 
+			case OP_TMAP2POLY:
```

Comment:
Consider refactoring this switch statement to make use of fallthrough instead of interlaced, identical subroutines